staging/src/k8s.io/pod-security-admission/check_dropAllCapabilities.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/pod-security-admission/api"
+)
+
+func init() {
+	addCheck(CheckDropCapabilities)
+}
+
+// CheckDropCapabilities returns a restricted level check
+// that ensures all capabilities are dropped in 1.22+
+func CheckDropCapabilities() Check {
+	return Check{
+		ID:    "dropCapabilities",
+		Level: api.LevelRestricted,
+		Versions: []VersionedCheck{
+			{
+				MinimumVersion: api.MajorMinorVersion(1, 22),
```

Comment:
Please refer to other `check_*.go` under `staging/src/k8s.io/pod-security-admission/policy`, pod security version and kubernetes release version are independent, preferably starting from 1.0